### DISCUSSION TITLE
Permissions on Page/ Frontend

### DISCUSSION THREAD

**Discussion by marvingoossens** [pimcore/pimcore#15935] [Q&A / Support] [answered]
I have implemented a controller and a template with a form to change and save a DataObject.
And I created a page under documents so that you can request the page under localhost/translator.

But now no login is needed for localhost/translator.
How can I restrict the page and only allow it for a specific user group?

Im very new to Pimcore, maybe you have some tipps or links where I can learn how to solve this problem.

We use Pimcore 10.5.

**Comment by NiklasBr** [comment]
Maybe this could be of use?

https://github.com/pimcore/frontend-permission-toolkit

**Reply by dvesh3** [reply, marked as answer]
look at these docs, might help in decision how to implement it:
https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Security_Authentication/Authenticate_Pimcore_Objects
https://pimcore.com/docs/platform/Pimcore/Best_Practice/Build_Role_Rights_System_for_Frontends
